backend/src/routes/api/routes/index.ts

```diff
@@ -0,0 +1,16 @@
+import { KubeFastifyInstance } from '../../../types';
+import { FastifyRequest } from 'fastify';
+import { secureRoute } from '../../../utils/route-security';
+import { getRoute } from './routesUtils';
+
+module.exports = async (fastify: KubeFastifyInstance) => {
+  fastify.get(
+    '/:namespace/:name',
+    secureRoute(fastify)(
+      async (request: FastifyRequest<{ Params: { namespace: string; name: string } }>) => {
+        const { namespace, name } = request.params;
+        return await getRoute(fastify, namespace, name);
```

Comment:
Curious -- we don't need a backend exposed route for getting Route objects. Why did we create this?

Reply:
Good question. I don't know either, I will get rid of it 😞